EnglishFilter/sections/specific.txt

```diff
@@ -3063,6 +3063,7 @@ cutpaid.com###link-view > center > iframe
 sozosblog.com###banner-lad
 sozosblog.com###load-featured-content
 sozosblog.com###sidebar a[href="/go.php"][rel="nofollow"] > img
+samehadaku.vip##.adsplayer > div.close-button
```

Comment:
It was fixed by `AdBlockID` (this site is Indonesian). You can fix here another elements like share `Bagikan`